docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -11,7 +11,7 @@ This page is an overview of Kubernetes.
 
 {% capture body %}
 Kubernetes is a portable, extensible open-source platform for managing
-containerized workloads and services, that facilitates both
+containerized workloads and services, this facilitates both
```

Comment:
Should instead be:

> containerized workloads and services that facilitates both

Reply:
thanks for your reply
i think i was wrong but the correct format should be :
containerized workloads and services that facilitate both